message/validation/validation.go

```diff
@@ -231,9 +231,19 @@ func (mv *messageValidator) ValidatorForTopic(_ string) func(ctx context.Context
 // Depending on the outcome, it will return one of the pubsub validation results (Accept, Ignore, or Reject).
 func (mv *messageValidator) ValidatePubsubMessage(_ context.Context, peerID peer.ID, pmsg *pubsub.Message) pubsub.ValidationResult {
 	if mv.selfAccept && peerID == mv.selfPID {
-		msg, _ := commons.DecodeNetworkMsg(pmsg.Data)
+		rawMsgPayload, _, _, err := commons.DecodeSignedSSVMessage(pmsg.Data)
+		if err != nil {
+			mv.logger.Error("decoding signed ssv message error", zap.Error(err))
```

Comment:
I think it would be better to keep error logs text consistent, we usually start them with `failed to ...`

```suggestion
			mv.logger.Error("failed to decode signed ssv message", zap.Error(err))
```

Reply:
fixed this. Thanks